src/clr-angular/layout/tabs/tab-content.spec.ts

```diff
@@ -14,6 +14,7 @@ import { ClrTabContent } from './tab-content';
 @Component({
   template: `
         <clr-tab-content>Content1</clr-tab-content>
+        <ng-container [ngTemplateOutlet]="tabContent.templateRef"></ng-container>
```

Comment:
I think we need a comment here _why_ this is here. Its here because we don't have a clr-tabs container which hosts this container for us.

Reply:
You're right. Even I needed some time to remember why we do it this way.